WeatherApp/WebAPI/Controllers/ErrorController.cs

```diff
@@ -0,0 +1,32 @@
+﻿using BL.Validation;
+using Microsoft.AspNetCore.Diagnostics;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.AspNetCore.Mvc.ModelBinding;
+
+namespace WebAPI.Controllers
+{
+    [ApiExplorerSettings(IgnoreApi = true)]
+    public class ErrorController : Controller
+    {
+        [HttpGet]
+        [Route("/Error")]
+        public IActionResult GetError()
```

Comment:
PLease return some class instead of IActionResult

Reply:
New ErrorResponse class is returned now.